### DISCUSSION TITLE
Control how long a peripheral job should take when using the loopback adapter

### DISCUSSION THREAD

**Discussion by jfaigan** [openTCS/opentcs#76] [Q&A] [answered]
Is there a way to indicate how long a peripheral job should take when using the loopback adapter?  

Basically, I am trying to simulate the time a vehicle should pause and wait at a location while a process is being completed. Once completed, I want the vehicle to continue processing the rest of the orders in the Transportation Order.

I have a workaround in mind where I can watch the events being generated and when the Location is being processed, I can use the API to Pause the Vehicle and after a certain amount of time has elapsed, I can Unpause the Vehicle.

**Comment by swltr** [comment, marked as answer]
For the loopback peripheral driver, the time it takes for a job to complete is currently not configurable. (For the loopback vehicle driver, it is possible to specify a single value that is used for all destination operations this vehicle performs - see [the User's Guide](https://www.opentcs.org/docs/5.7/user/opentcs-users-guide.html#_configuring_a_virtual_vehicles_characteristics).)

**Reply by jfaigan** [reply]
Ok, that is what I thought.  Thanks for the info.
